src/common/common.py

```
@@ -41,6 +41,11 @@ def load_params(default: bool = False) -> dict[str, Any]:
     Returns:
         dict[str, Any]: A dictionary containing the parameters.
     """
+    
+    # Check if workspace is enabled. If not, load default parameters.
+    if not st.session_state.settings["is_workspace_enabled"] == True:
```

Ruff (0.8.2)

Use `st.session_state.settings["is_workspace_enabled"] != True` instead of `not st.session_state.settings["is_workspace_enabled"] == True`

Replace with `!=` operator

(SIM201)